mimic/rest/maas_api.py

```diff
@@ -1299,6 +1304,31 @@ def multiplot(self, request, tenant_id):
         self._audit('rollups', request, tenant_id, status, content)
         return json.dumps({'metrics': metrics_replydata})
 
+    @app.route('/v1.0/<string:tenant_id>/views/latest_alarm_states', methods=['GET'])
+    def latest_alarm_states(self, request, tenant_id):
+        """
+        Gets entities grouped with their latest alarm states.
```

Comment:
Can we add a link to the docs for this api, if one exists.

Same for the one below.